source_function.py

```diff
@@ -0,0 +1,12 @@
+import math
+
+def special_function_for_first_time_step(u, t, user_input):
+   
+    s1 = floor(user_input['Time Steps']/user_input['frequency'])
```

Comment:
Poor variable name choice - what does this mean?  Has units of time squared?

Reply:
* There was a minor error in naming the variables. That should have been number of temporal nodes divided by the frequency. 

* Has units Time. Again, this confusion was caused due to wrong input keys.